### DISCUSSION TITLE
Display selection mode doesent work.

### DISCUSSION THREAD

**Discussion by crydotsnake** [JetBrains/ideavim#739] [Q&A]
Hello!

I've set:

```
set showmode
```

In my `.ideavimrc` config file. But after a restart, i still cant see the current selection mode.

I 'm using PHPStorm PhpStorm 2023.2.3 with the new UI enabled.

**Comment by citizenmatt** [comment]
The new UI hides status text by default, positioning the breadcrumbs there instead. You can right click on the status bar and status icons and choose to show breadcrumbs or status text.
